service/operator/service_test.go

```diff
@@ -1,95 +1,604 @@
 package operator
 
 import (
+	"reflect"
 	"testing"
 
+	"github.com/giantswarm/ingresstpr"
+	"github.com/giantswarm/ingresstpr/guestcluster"
+	"github.com/giantswarm/ingresstpr/hostcluster"
+	"github.com/giantswarm/ingresstpr/hostcluster/ingresscontroller"
+	"github.com/giantswarm/ingresstpr/protocolport"
 	apiv1 "k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/pkg/util/intstr"
 )
 
-func Test_Service_newClusterPort(t *testing.T) {
+func Test_Service_GetDesiredState(t *testing.T) {
```

Comment:
I do this quite often. I try to structure the test name that way to make it also more readable. At least it is an allowed practise. See https://golang.org/pkg/testing/.